### PR TITLE
XW-2593 | Preprender categories module on articles and categories

### PR DESCRIPTION
**Pull request by kvas-damian**
## Links
* https://wikia-inc.atlassian.net/browse/XW-2593

## Description
Content rendered on server side is more attractive for Search Engine crawlers. This PR introduce prerendering categories module on article and category pages

## Reviewers
ping @Wikia/x-wing

### CODE REVIEW

File: server/app/views/_partials/article-footer/categories.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<nav class="collapsible-menu">
```
Comment (rogatty) [inline server/app/views/_partials/article-footer/categories.hbs]: Are these the same as on front-end? Please add a comment if yes (also backlinks).

Comment (kvas-damian) [inline server/app/views/_partials/article-footer/categories.hbs]: Done

File: server/app/views/_partials/article-footer/categories.hbs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{{! Keep the same HTML structure as in front/main/app/templates/components/article-wrapper.hbs }}
+<nav class="collapsible-menu">
+	<div class="collapsed title">
```
Comment (hakubo) [inline server/app/views/_partials/article-footer/categories.hbs]: since they are collapsed - I imagine they are displayed: none - would anything index it?

Comment (kvas-damian) [inline server/app/views/_partials/article-footer/categories.hbs]: It's mobile view. it's acceptable that some content is hidden by default but accessible in UI.

**Pull request merged**